packages/dds/tree/src/test/memory/tree.spec.ts

```diff
@@ -33,28 +39,65 @@ class RootNodeSchema extends builder.object("root-item", {
 	child: builder.optional(ChildNodeSchema),
 }) {}
 
-function createLocalSharedTree(id: string): TreeView<typeof RootNodeSchema> {
-	const sharedTree = SharedTree.create(
+const initialState = {
+	child: {
+		propertyOne: 128,
+		propertyTwo: {
+			itemOne: "",
+		},
+		propertyThree: new Map([["numberOne", 1]]),
+	},
+};
+
+// Array has 2 allowable types to make it less efficient for uniform chunks.
+class RootNodeSchemaBasicChunks extends builder.array("root-item-with-basic-chunks", [
```

Comment:
It might be more clear if you name this after what it is, rather than how its used. Ex: PolymorphicArray.

Then if other code wants to use it (ex: not at the root) its name doesn't suddenly become wrong, and code using it know why it makes basic chunks (because its polymorphic).

RootNodeSchemaUniform beelow could then become MonomorphicArray to match.

Reply:
Updated with suggested names